google/cloud/spanner_v1/pool.py

```diff
@@ -126,6 +138,19 @@ def _new_session(self):
             labels=self.labels, database_role=self.database_role
         )
 
+    def _format_trace(self, frames: List[traceback.FrameSummary]) -> str:
+        """Helper for formatting traceback frames into more readable trace.
+        :type frames: list[traceback.FrameSummary]
+        :param frames: list of stack frames
+        :rtype: str
+        :returns: fromatted trace.
+        """
+        stack = [
```

Comment:
Can we format this in the python format and add a test case for it?

Reply:
Removed the custom method and using the method available in traceback library.